queries/c_sharp/highlights.scm

```diff
@@ -1,5 +1,9 @@
 (identifier) @variable
 
+((identifier) @keyword @_p
+  (#eq? @_p "value")
```

Comment:
You can use `@keyword` instead of creating a new caputre

Reply:
`value` is the contextual keyword (see [more](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/value))
For example:
```CSharp
class Foo
{
    private int number;
    public int Property
    {
        get => number;
        set {number = value;} // "value" is keyword
    }

    public void Bar()
    {
        var value = 10;
        var another = value; // "value" is simple variable name
    }
}
```

Therefore it should be highlighted when placed in the `accessor_declaration` subtree.

Or the comment is about simplifying the query and am I missing something?

Reply:
Just found out that `get`, `set`, `add`, `remove` and `init` are also valid names for variables. As well as `and` and `or` and `record`

Reply:
what I mean is that you can write this query as

```
((identifier) @keyword
  (#eq? @keyword "value")
```

there is no need to create `@_p`

Reply:
Got it! Removed redundant `@_p` capture.

Also checked contextual keywords highlighting (wrote about them earlier) and they are all good. Variables like `set` are highlighted properly.